R/escape.R

```diff
@@ -108,6 +108,11 @@ escape.integer64 <- function(x, parens = NA, collapse = ", ", con = NULL) {
   sql_vector(x, parens, collapse, con = con)
 }
 
+#' @export
+escape.raw <- function(x, parens = NA, collapse = ", ", con = NULL) {
```

Comment:
This feels a bit weird to me because x is going to be a vector, but this function will always return a single value. That makes it different to the other methods.

Reply:
It does, but raw is actually different from other types.